resource/ckb.toml

```diff
@@ -76,6 +76,8 @@ bootnodes = [] # {{
 # whitelist_only = false
 ### Whitelist peers connecting from the given IP addresses
 # whitelist_peers = []
+### enable reuse port, default enable on Linux, not supported on other OS yet
+# reuse = false
```

Comment:
```suggestion
### Try to enable port reuse when the hint is true. Currently, port reuse will only be enabled on Linux.
# reuse_hint = false
```

Reply:
If it's system specific, let user configure it separately.

```suggestion
### Enable to reuse port on Linux
# reuse_port_on_linux = false
```

Reply:
The term "reuse" is ambiguous. I suggest using the full name "reuseport", "enabled_reuseport" or other clear names.

Also, the comment should include information about `SO_REUSEPORT`.